### PR TITLE
added media events that do not bubble to list eventTypesThatDontBubble

### PR DESCRIPTION
**Pull request by iambumblehead**

### CODE REVIEW

File: src/render-dom.js [hunk 1]
```diff
@@ -117,6 +117,24 @@ const eventTypesThatDontBubble = [
...
+  `waiting`,
+  `seeking`,
+  `seeked`,
+  `ended`,
+  `loadedmetadata`,
+  `loadeddata`,
+  `canplay`,
+  `canplaythrough`,
+  `durationchange`,
+  `timeupdate`,
```
Comment (staltz) [inline src/render-dom.js]: `timeupdate` is duplicated. Can you remove?

Comment (iambumblehead) [inline src/render-dom.js]: the duplicate is removed

File: src/render-dom.js [hunk 2]
```diff
@@ -117,6 +117,23 @@ const eventTypesThatDontBubble = [
...
+  `canplay`,
+  `canplaythrough`,
+  `durationchange`,
+  `play`,
+  `pause`,
+  `ratechange`,
+  `volumechange`,
+  `suspend`,
+  `emptied`,
+  `stalled`
```
Comment (staltz) [inline src/render-dom.js]: Can you add a comma here? To adhere to our eslint convention

Comment (iambumblehead) [inline src/render-dom.js]: the comma is added

**Comment by staltz** [comment]
Thanks!

**Pull request merged**